assets/schema_input.json

```diff
@@ -18,6 +18,26 @@
                 "errorMessage": "Where the references came from",
                 "meta": ["source"]
             },
+            "source_dbsnp": {
+                "type": "string",
+                "errorMessage": "Where the references came from",
```

Comment:
What does that mean exactly? A repository on GitHub? An organisation? A URL?

Reply:
it's a free string, but I would expect just a tiny string, I'll write docs and error message and help later on